pumps/mongo.go

```diff
@@ -598,8 +598,8 @@ func (m *MongoPump) WriteUptimeData(data []interface{}) {
 		decoded := analytics.UptimeReportData{}
 
 		if err := msgpack.Unmarshal([]byte(v.(string)), &decoded); err != nil {
+			// ToDo: should this work with serializer?
```

Comment:
@matiasinsaurralde @tbuchaillot

Reply:
Good point @sredxny. Probably is better to have consistency between uptime analytics and common analytics. So yeah, we need to implement something at pump and Gateway level for uptime analytics.